sound/soc/sof/topology.c

```diff
@@ -2497,7 +2514,7 @@ static int sof_widget_unload(struct snd_soc_component *scomp,
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
 	const struct snd_kcontrol_new *kc;
 	struct snd_soc_dapm_widget *widget;
-	struct sof_ipc_pipe_new *pipeline;
+	const struct sof_ipc_pipe_new *pipeline;
```

Comment:
@lyakh This is breaking the xmas tree.

Reply:
@kv2019i I don't think it's a hard rule, more of a soft recommendation. It's also already broken between the previous 2 lines.